### ISSUE TITLE
Уведомления на почту

### ISSUE THREAD

**Issue by yupe** [brussens/yupe-message#24]
Я бы сделал уведомлялку на почту о новом сообщении.

**Comment by brussens** [comment]
@yupe будет к RC

**Comment by ApexWire** [comment]
я бы добавил сюда еще уведомления о непрочитанных сообщениях
и лучше бы эти уведомления реализовать используя команду, которую можно будет запускать по крону

**Comment by brussens** [comment]
@ApexWire я не против, просто не хочется писать отдельно для модуля такое. Думаю будет не плохо, если это будет в самой системе и работать с этим можно будет по API

**Comment by ApexWire** [comment]
Я думал реализовать это с помощью модуля уведомлений (notify).
Схема примерно следующая:
- у модуля уведомлений есть таблица в которую складываются все уведомления и по крону они отправляются.
- каждое уведомление содержит следующие поля: тип отправки (email, sms), куда (зависит от типа почта, номер телефона), текст уведомления.

Но сначала надо будет изучить модуль уведомлений)

**Comment by brussens** [comment]
@ApexWire да, с модулем знакомлюсь по мере возможного. Думаю в ближайшее время сделаю.
